### PR TITLE
logstash integration test on based official logstash-input-beats plugin

### PR DESCRIPTION
**Pull request by urso**

### CODE REVIEW

File: scripts/docker/logstash/logstash.conf.2.tmpl [hunk 1]
```diff
@@ -0,0 +1,16 @@
+input {
+  beats {
+    port => 12345
```
Comment (ruflin) [inline scripts/docker/logstash/logstash.conf.2.tmpl]: should we use 5044 here too?

Comment (urso) [inline scripts/docker/logstash/logstash.conf.2.tmpl]: let's do the port update in another PR. #169 already includes some changes. Missing ports will be updated after merging #168 and #169.
